tests/rbd_mirror/test_rbd_image_meta_mirroring.py

```diff
@@ -0,0 +1,102 @@
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+import pdb
```

Comment:
Could we avoid debugging imports?

Reply:
Sorry, that was a mishap, I've removed them.